pages/game.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+
+import CustomHead from '../src/components/head';
+import Button from '../src/components/button';
+import Nav from '../src/components/nav';
+import Grid from '../src/components/grid';
+import ActivityIndicator from '../src/components/activityIndicator';
+
+import { buildGrid } from '../src/core/game';
+
+export default class Index extends Component {
+    handleClickSingleGame = () => {};
+
+    state = {
+        isLoading: true,
+        currentGrid: null,
+        initialGrid: null,
+        turn: -1,
+        isVictory: false,
+    };
+
+    renderWinnerMessage(isVictory, turn) {
```

Comment:
avoid renderXXX methods. Just put the condition directly inside the rendered JSX or extract in another component.